scripts/add_chats.py

```diff
@@ -1,94 +1,125 @@
 import datetime
 import json
+import os
 
 import httpx
+from ragna.core._rag import default_user
 
 
 def main():
-    url = "http://127.0.0.1:31476"
-    user = "Ragna"
+    client = httpx.Client(base_url="http://127.0.0.1:31476")
+    client.get("/").raise_for_status()
 
-    client = httpx.Client(base_url=url, headers={"X-User": user})
-    client.get("").raise_for_status()
+    ## authentication
```

Comment:
This is the only substantial change in this file. The remaining changes are just better error handling.